sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/HttpPipelineHelper.cs

```diff
@@ -207,15 +207,21 @@ internal static ExportResult HandleFailures(HttpMessage httpMessage, PersistentB
                 }
             }
 
-            AzureMonitorExporterEventSource.Log.TransmissionFailed(fromStorage: false, statusCode: statusCode, connectionVars: connectionVars, requestEndpoint: httpMessage.Request.Uri.Host, willRetry: (result == ExportResult.Success));
+            AzureMonitorExporterEventSource.Log.TransmissionFailed(
```

Comment:
What is the cost of logging a message when a listener is enabled and the batch fail all the times?

Reply:
Is it possible to asynchronously log this message?

Reply:
> what is the cost

If this is happening repeatedly, the Exporter should backoff.

> is it possible to asynchronously log

I don't know, i'll look into this :)

Reply:
> If this is happening repeatedly, the Exporter should backoff.

May not be true, if its a partial failure.

Reply:
mitigating this by only deserializing errors when LogLevel is set to Verbose